File: src/main/java/kr/nyamnyam/model/domain/NoticeModel.java
package kr.nyamnyam.model.domain;

import kr.nyamnyam.model.entity.NoticeEntity;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

import java.util.Date;

@Component
@Data
@Builder
@NoArgsConstructor
public class NoticeModel {
    private Long id;
    private String title;
    private String content;
    private Long hits;
    private Date date;

    @Builder
    public NoticeModel(Long id, String title, String content, Long hits, Date date) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.date = date;
        this.hits = hits;
    }

}

File: src/app/model/notice.model.ts
export interface NoticeModel {
  id: number;  
  title: string;
  content: string;
  date: string;
  hits: number;
}

export const initialNotice: NoticeModel = {
  id: 0,
  title: '',
  content: '',
  date: '',
  hits: 0
}

